selectize/index.d.ts

```diff
@@ -155,7 +155,7 @@ declare namespace Selectize {
          *
          * Default: null
          */
-        plugins?: string[];
+        plugins?: string[] | Object;
```

Comment:
please be more specific and define this interface. `Object` is too broad of a type.